outputs/aws.go

```diff
@@ -135,6 +138,37 @@ func (c *Client) SendMessage(falcopayload types.FalcoPayload) {
 	c.PromStats.Outputs.With(map[string]string{"destination": "awssqs", "status": "ok"}).Inc()
 }
 
+// UploadS3 upload payload to S3
+func (c *Client) UploadS3(falcopayload types.FalcoPayload) {
```

Comment:
You need to declare the new client in https://github.com/falcosecurity/falcosidekick/blob/master/main.go

You need to declare the handler in https://github.com/falcosecurity/falcosidekick/blob/master/handlers.go